### PR TITLE
Update R2X function and figure1

### PR DESCRIPTION
**Pull request by murphymadeleine21**
Let me know if this is what you had in mind.

### CODE REVIEW

**Comment by murphymadeleine21** [comment]
Looks like it's having some issues passing the test that insures R2X increases with component size. Also I noticed that the tensor R2X didn't seem to change before & after PCA.

File: syserol/figures/figure1.py [hunk 1]
```diff
@@ -15,11 +15,14 @@ def makeFigure():
 
     cube, glyCube = createCube()
     comps = np.arange(1, 13)
-    arr = [perform_CMTF(cube, glyCube, i)[2] for i in comps]
+    tensorArr = [perform_CMTF(cube, glyCube, i)[2] for i in comps]
```
Comment (aarmey) [inline syserol/figures/figure1.py]: You don't want to do this, as you're running the same factorization twice. Write a loop to save both of these from one factorization run.

**Review by aarmey — CHANGES_REQUESTED** [review]
Right—PCA only adds to the variance explained within the matrix. For the test, it can just look at the tensor R2X.

**Comment by murphymadeleine21** [comment]
Oh okay right.

**Review by aarmey — APPROVED** [review]

**Pull request merged**